src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+:root {
+  --header-height: 60px;
+  --underline-height: 4px;
+  --horizontal-padding: 50px;
+  --hover-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: calc(100% - 2 * var(--horizontal-padding));
+  background: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 var(--horizontal-padding);
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__images {
+  margin-top: 10px;
+}
+
+.header__nav {
+  height: var(--header-height);
+}
+
+.nav__list {
+  text-transform: uppercase;
+  list-style-type: none;
+  display: flex;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  cursor: pointer;
+}
+
+.nav__item:nth-child(n + 2) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  width: 100%;
```

Comment:
Also here you dont need to set width for elements, as they will cover needed space

Reply:
Thank you. One more time :). Changes added.